frontend/src/utils/http-methods.ts

```diff
@@ -49,5 +49,3 @@ class HttpMethods {
     return await this.HTTP<T>(url, init);
   }
 }
-
-export const HTTP = new HttpMethods();
```

Comment:
restore this export and try to work with composition and use the HTTP instance instead of inheritance